jscomp/syntax/ast_external_attributes.ml

```diff
@@ -555,7 +555,7 @@ let handle_attributes
           left_attrs
         end
 
-      | _ -> Location.raise_errorf ~loc "conflict attributes found [@@bs.obj]"  
+      | _ -> Location.raise_errorf ~loc "Attribute found that conflicts with [@@bs.obj]"  
```

Comment:
Wouldn't it be useful to at least tell the user that one of the conflicting attributes is `[@@bs.obj]`? That could help with troubleshooting the problem, even if we don't say which other attribute conflicts.